lib/net/ldap/entry.rb

```diff
@@ -196,4 +196,8 @@ def setter?(sym)
     sym.to_s[-1] == ?=
   end
   private :setter?
+
+  def ==(other)
+    return other.instance_of?(self.class) && @myhash == other.to_h
```

Comment:
We don't need an explicit return here.
```suggestion
    other.instance_of?(self.class) && @myhash == other.to_h
```